cluster-autoscaler/cloudprovider/azure/azure_client.go

```diff
@@ -206,13 +207,24 @@ func newServicePrincipalTokenFromCredentials(config *Config, env *azure.Environm
 	return nil, fmt.Errorf("no credentials provided for AAD application %s", config.AADClientID)
 }
 
+func newAuthorizerFromCredentials(config *Config, env *azure.Environment) (autorest.Authorizer, error) {
```

Comment:
nit (naming): with CLI support, that's not necessarily `FromCredentials` anymore right?

Reply:
I removed `FromCredentials`.